proseco/guide.py

```diff
@@ -623,17 +622,20 @@ def check_mag_spoilers(cand_stars, ok, stars, n_sigma):
 
     mag_spoiled = np.zeros(len(ok)).astype(bool)
     rej = []
-    for cand in cand_stars[ok]:
-        pix_dist = np.sqrt(((cand['row'] - stars['row']) ** 2) +
```

Comment:
Don't need to calculate the distance to every star, only stars that are in the spoiler box.  (Right?)

Reply:
Sure.  That seems right.  I'd tried to optimize by just not calling the full routine much, but plenty of room to improve it locally.  Thanks!